.gitattributes

```diff
@@ -1,9 +1,14 @@
-/tests export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.scrutinizer.yml export-ignore
-.travis.install.sh export-ignore
-.travis.yml export-ignore
-phpmd.xml.dist export-ignore
-phpunit.xml.dist export-ignore
-composer.lock export-ignore
+# Exclude build/test files from archive
+/.github          export-ignore
+/tests            export-ignore
+.gitattributes    export-ignore
+.gitignore        export-ignore
+phpbench.json     export-ignore
+phpcs.xml.dist    export-ignore
+phpmd.xml.dist    export-ignore
+phpstan.neon.dist export-ignore
+phpunit.xml.dist  export-ignore
+composer.lock     export-ignore
+
+# Configure diff output for .php files.
+*.php diff=php
```

Comment:
I think this part should be done by contributors globally on every machine they use, rather than by every Doctrine project.

```suggestion
```

Reply:
Thank you, I removed it.